cypress/pageobjects/virtualmachine.po.ts

```diff
@@ -0,0 +1,68 @@
+import { Constants } from "../constants/constants";
+import LabeledSelectPo from './components/labeled-select.po';
+import LabeledInputPo from './components/labeled-input.po';
+const constants = new Constants();
+
+export class VmsPage {
+  public goToList() {
+    cy.visit(constants.vmPage);
+    cy.intercept('GET', '/v1/harvester/kubevirt.io.virtualmachines').as('loadVM');
+    cy.wait('@loadVM')
+    expect(cy.url().should('eq', constants.vmPage));
+  }
+
+  public goToCreate() {
+    this.goToList()
+    cy.contains('Create').click()
+    cy.intercept('GET', '/v1/harvester/harvesterhci.io.virtualmachineimages').as('loadVMCreate');
+    cy.wait('@loadVMCreate')
+  }
+
+  public setDefaultValue() {
+    this.namespace().select('default')
+    this.name().input('test-vm-name-automation')
+    this.description().input('test-vm-description-automation')
+    this.cpu().input('2')
+    this.memory().input('4')
```

Comment:
These should probably be accepted as parameters from calling the method in the test spec not statically set.